### ISSUE TITLE
[ Option Request ] Readd cookies to Clear-Site-Data Header

### ISSUE THREAD

**Issue by Ornias1993** [nextcloud/server#19749]
Previously there was a feature (or was it a bug really...) that lead to Nextcloud logout also deleting all cookies for said domain. This was not great (understatement) for many users.

However:
Some of us would actually have a use for such a feature. For example some of us have a few extra webapps included as an iframe within nextcloud and want to make sure nextcloud logout is also logging out uses that forgot to logout seperately.

yes there are other solutions for this. but for some this is a relatively simply solution to implement as a config.php flag which just covers out usecase.

The usecase for this is increased by the weird lack of custom logout scripts on nextcloud (unless you go modify the source and have changes lost on update)...

In short: I think the full deletion of this feature/bug was just as overzealous as the way it was included in the first place. Giving us users the option to have this behavior, would be a good thing for everyone! :)

**Comment by szaimen** [comment]
Hi, please see the following config options https://docs.nextcloud.com/server/20/admin_manual/configuration_server/config_sample_php_parameters.html
![image](https://user-images.githubusercontent.com/42591237/120174341-4436d000-c205-11eb-9755-494356f7103a.png)

**Comment by Ornias1993** [comment]
@szaimen Thats not what I was talking about at all.
I was talking about logout nuking ALL cookies (not only those from nextcloud) for it's domain.

Which I quite thoroughly explained.
